Sources/FeaturevisorSDK/Emitter.swift

```diff
@@ -3,6 +3,7 @@ public enum EventName {
     case refresh
     case update
     case activation
+    case datafileFetchError
```

Comment:
What do you think? Is it fine or should be with "_" so `_datafileFetchError`?

Reply:
let's go with `_` prefix, so we don't forget we have to revisit it here as well. not sure if Swift will allow it 😬

Reply:
Swift is cool :) - done